source/patterns/@aws-solutions-constructs/aws-alb-fargate/README.md

```diff
@@ -24,42 +24,59 @@
 
 This AWS Solutions Construct implements an an Application Load Balancer to an AWS Fargate service
 
-Here is a minimal deployable pattern definition in Typescript:
+Here is a minimal deployable pattern definition:
 
+Typescript
 ``` typescript
-  import { AlbToFargate, AlbToFargateProps } from '@aws-solutions-constructs/aws-alb-fargate';
+import { AlbToFargate, AlbToFargateProps } from '@aws-solutions-constructs/aws-alb-fargate';
+import * as acm from 'aws-cdk-lib/aws-certificatemanager';
 
-  // Obtain a pre-existing certificate from your account
-  const certificate = acm.Certificate.fromCertificateArn(
-        scope,
-        'existing-cert',
-        "arn:aws:acm:us-east-1:123456789012:certificate/11112222-3333-1234-1234-123456789012"
-      );
+// Obtain a pre-existing certificate from your account
+const certificate = acm.Certificate.fromCertificateArn(
+    this,
+    'existing-cert',
+    "arn:aws:acm:us-east-1:123456789012:certificate/11112222-3333-1234-1234-123456789012"
+);
 
-  const props: AlbToFargateProps = {
+const props: AlbToFargateProps = {
     ecrRepositoryArn: "arn:aws:ecr:us-east-1:123456789012:repository/your-ecr-repo",
     ecrImageVersion: "latest",
     listenerProps: {
-      certificates: [ certificate ]
+        certificates: [certificate]
     },
     publicApi: true
-  };
+};
 
-  new AlbToFargate(stack, 'new-construct', props);
+new AlbToFargate(this, 'new-construct', props);
 ```
 
-## Initializer
-
-``` text
-new AlbToFargate(scope: Construct, id: string, props: AlbToFargateProps);
+Python
+``` python
+from aws_solutions_constructs.aws_alb_fargate import AlbToFargate, AlbToFargateProps
+from aws_cdk import (
+    aws_certificatemanager as acm,
+    aws_elasticloadbalancingv2 as alb
+)
+
+# Obtain a pre-existing certificate from your account
+certificate = acm.Certificate.from_certificate_arn(
+      self,
```

Comment:
This indent doesn't match the indent in the next function call (line 68). I know that indentions are important in Python ([sarcasm]one of the things I love so much about it[/sarcasm]), we should be consistent.